static/bidder-info/mgidX.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://us-east-x.mgid.com/pserver"
+endpoint: "https://{{.Host}}.mgid.com/pserver"
```

Comment:
@xmgiddev 

should update json test to include endpoint from `bidder-info`
https://github.com/prebid/prebid-server/blob/ec729e6e5d50a1306d76332fda52108b9d313c1d/adapters/mgidX/mgidX_test.go#L12-L20